lib/Doctrine/ORM/Configuration.php

```diff
@@ -46,12 +57,16 @@ class Configuration extends \Doctrine\DBAL\Configuration
     public function setProxyDir($dir)
     {
         $this->attributes['proxyDir'] = $dir;
+
+        $this->getProxyManagerConfiguration()->setProxiesTargetDir($dir);
     }
 
     /**
      * Gets the directory where Doctrine generates any necessary proxy class files.
      *
      * @return string|null
+     *
+     * @deprecated please do not use this anymore
```

Comment:
Should we drop this?

Reply:
Not removed: too commonly used, therefore keeping it and documented as deprecated

Reply:
Drop it! Unless you can make it fully functional to keep BC.
I didn't for Metadata, so I'm assuming if people will already change for Metadata, they could do for Proxy also.

Reply:
I kept it because this is a high visibility endpoint: most integration layers with the ORM touch the configuration object. I will do a github search before considering a droo